tests/test_meta.py

```diff
@@ -0,0 +1,41 @@
+##############################################################################
+# blosc2_grok: Grok (JPEG2000 codec) plugin for Blosc2
+#
+# Copyright (c) 2023  The Blosc Development Team <blosc@blosc.org>
+# https://blosc.org
+# License: GNU Affero General Public License v3.0 (see LICENSE.txt)
+##############################################################################
+
+import numpy as np
+import pytest
+from PIL import Image
+from pathlib import Path
+
+
+import blosc2
+
+project_dir = Path(__file__).parent.parent
+@pytest.mark.parametrize('image', [project_dir / 'examples/kodim23.png', project_dir / 'examples/MI04_020751.tif'])
+@pytest.mark.parametrize('meta', [1 * 10, 2 * 10, 5 * 10, 8 * 10, 10 * 10])
```

Comment:
Can you add a test for 20 * 10?  I'd like to see that 200 is not (wrongly) interpreted as negative (128 - 200 = -72).